kernel: userspace: fix typo in the comments

naiyuantian opened the pull request:
While reading the code, found some  typos in the code comments, line 226 and 668.
Fix comments to make it more solid.

Signed-off-by: Naiyuan Tian <naiyuan.tian@intel.com>

nashif commented:
Please fix your Author details, right now it shows:

Author: naiyuantian <naiyuan.tian@intel.com>

naiyuantian committed:
kernel: userspace: fix typo in the comments

While reading the code, found some typos in the code comments,
line 226 and 668.
Fix comments to make it more solid.

Signed-off-by: Naiyuan Tian <naiyuan.tian@intel.com>

ceolin reviewed (approved):
Line references are mostly useless since they change frequently. It is preferred a more descriptive text
